filip/models/ngsi_ld/context.py

```diff
@@ -35,14 +37,60 @@ class ContextProperty(BaseModel):
         >>> attr = ContextProperty(**data)
```

Comment:
@LaraRoth seems that there is some unsolved conflict here

Reply:
I  believe the ContextProperty Model is missing the parameters 'createdAt' and 'modifiedAt' which will be sent back by the context broker when the option 'sysAttrs' is used for getting an entity from the broker.